clients/js/src/examples/multisig.ts

```diff
@@ -11,16 +11,16 @@ import {
   signTransactionMessageWithSigners,
 } from '@solana/kit';
 import { TOKEN_2022_PROGRAM_ADDRESS } from '@solana-program/token-2022';
-import { findWrappedMintAuthorityPda, findWrappedMintPda } from '../generated';
-import { multisigOfflineSignWrapTx } from '../wrap';
 import {
+  findWrappedMintAuthorityPda,
+  findWrappedMintPda,
+  multisigOfflineSignWrapTx,
   combinedMultisigTx,
   createEscrowAccountTx,
-  createTokenAccountTx,
-  getOwnerFromAccount,
-} from '../utilities';
-import { createMintTx } from '../create-mint';
-import { multisigOfflineSignUnwrap } from '../unwrap';
+  createMintTx,
+  multisigOfflineSignUnwrap,
+} from '../index';
```

Comment:
In my separate testing, I copied/pasted this file over. I thought it would be best in these examples to be a bit more explicit and import from the entrypoint that will be exposed publicly (so as to catch any issues if we didn't export correctly).

The below utils are fine as they are just specific to this example.